corehq/db/config.yml

```diff
@@ -0,0 +1,20 @@
+# HQ database configuration for multiple database. This gets mapped to Django's DATABASES dict
```

Comment:
Yea 100%. @millerdev I'd also suggest you take a look at the localsettings template in [ansbile](https://github.com/dimagi/commcarehq-ansible/blob/master/ansible/roles/commcarehq/templates/localsettings.py.j2)